README.md

```diff
@@ -30,13 +30,15 @@ This is a very early version of the project.  Howerver, here is a list of things
 ## Build Instructions
 The following instructions is to build the code from `github`.The project uses the `GoDep` for dependency management.
 ```
-$ cd <go-workspace>/src/
-$ mkdir -p github.com/mesos
-$ cd github.com/mesos
+$ export GOPATH=<go-workspace>
```

Comment:
I think this will end up confusing users, on the README, under the requirements, we have `A standard and working Go workspace setup`

This would mean the user is expected to already have a working `GOPATH`, if the user is advance enough to have multiple `GOPATH`, they won't even need to read build instructions.